FinSearchUnified/Bundle/ProductNumberSearch.php

```diff
@@ -68,33 +68,33 @@ public function search(Criteria $criteria, ShopContextInterface $context)
         $response = $query->execute();
 
         if (empty($response)) {
-            self::setFallbackFlag(1);
+            self::setFallbackSearchFlag(1);
```

Comment:
In the implementation I wrote "Instead of the old implementation do", but please still set the fallback flag to `1` (like before).